src/content/structured/get-started/testing-with-cypress.mdx

```diff
@@ -73,115 +118,8 @@ module.exports = defineConfig({
 
 This allows traversing the shadow DOM using assertions without having to explicitly chain the `.shadow()` method.
 
-## Example: Component testing
-
-Below are examples of component tests on the `@ukic/react` components `IcTextfield` and `IcButton`. Component tests are normally located adjacent to the component they are testing and are rendered on a development server.
+## Example: End-to-end Testing
 
-```tsx
-// Textfield.tsx
-import { IcTextField, IcButton } from "@ukic/react";
-import React, { useState } from "react";
+Below is an example for end-to-end tests on a test-app. End-to-end tests normally live within the Cypress test directory and are rendered as a complete site with live interaction.
 
```

Comment:
Not sure if it would still be helpful to show component testing examples on the Cypress page.